elastic/kibana
# [Infra] Add 'Show in Hosts list' button in for host anomaly drop-down

Closes #174856

## Summary

This PR adds new 'Show affected Hosts' button in host anomaly drop-down in ML anomaly detection fly out. It applies `host.name` filer in hosts list and the same date range that user selected in ML fly out.

## Testing

1. Go to Inventory or Hosts page 
2. Click ML anomaly detection button at the top
3. Select Anomaly tab from the fly out
4. Change date range to make sure it applies to Hosts page
5. Click three dots button (more actions) and click 'Show affected Hosts'
6. User is taken to Hosts page, `host.name` filters are applied to display relevant hosts and the same date range as in ML fly out is applied

![Screen Recording 2024-08-30 at 12 33 36](https://github.com/user-attachments/assets/f8397d11-8fae-4f27-82ec-5b28da40db14)
